file_paths.md

```diff
@@ -0,0 +1,123 @@
+Puppet Installation Layout
+==========================
+
+This table specifies the file paths in a Puppet installation and the corresponding settings, denoted as `:setting`. The 3.x column shows the difference of the specification vs the actual 3.x implementation.
```

Comment:
Drop "actual"? That word implies a described-vs-actual comparison to me, but the comparison is 4.x/AIO vs 3.x.